Add relabelings in controller-servicemonitor.yaml

danksim opened the pull request:
<!--- Provide a general summary of your changes in the Title above --->
<!--- Please don't @-mention people in PR or commit messages (do so in an additional comment). --->
In order to drop metrics before ingestion, I need to utilize `relabelings` (before ingestion), not `metricRelabelings` (after ingestion)

## What this PR does / why we need it:
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
There were no ways to modify `relabeings` in the helm manifest before this PR

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation only

## Which issue/s this PR fixes
<!--
(optional, in `fixes #<issue number>` format, will close that issue when PR gets merged):

fixes #
-->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/kubernetes/ingress-nginx/blob/main/CONTRIBUTING.md) guide
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

cpanato commented:
@danksim please sign the CLA

cpanato reviewed (changes requested):
also please bump the chart version and add an entry in the changelog

danksim commented:
Thanks. I have addressed your comment and just signed the CLA, could you kindly confirm? Looks like the `cla/linuxfoundation` check is still failing @cpanato

cpanato commented:
> Thanks. I have addressed your comment and just signed the CLA, could you kindly confirm? Looks like the `cla/linuxfoundation` check is still failing @cpanato

we are still required to sign the other CLA as well, the new CLA is not fully migrated

cpanato commented:
/triage accepted
/priority important-soon
/kind feature

cpanato commented:
/check-cla

danksim commented:
/check-cla

cpanato commented:
/ok-to-test

cpanato reviewed (approved):
/approve

cpanato commented:
/hold for ci

cpanato commented:
/hold cancel

rikatz commented:
/lgtm
Thanks!

danksim commented:
@cpanato @rikatz thank you for the quick turnaround 🙌
